package.json

```diff
@@ -134,5 +136,11 @@
     "webpack": "1.12.9",
     "webpack-cli": "1.0.0",
     "webpack-dev-server": "1.14.0"
+  },
+  "peerDependencies": {
+    "babel-polyfill": "6.9.0",
+    "babel-runtime": "6.9.0",
+    "react": "0.14.7",
+    "react-dom": "0.14.7"
```

Comment:
Shouldn't this be `0.14.x` or something more lenient?

From https://nodejs.org/en/blog/npm/peer-dependencies/

> One piece of advice: peer dependency requirements, unlike those for regular dependencies, should be lenient. You should not lock your peer dependencies down to specific patch versions. It would be really annoying if one Chai plugin peer-depended on Chai 1.4.1, while another depended on Chai 1.5.0, simply because the authors were lazy and didn't spend the time figuring out the actual minimum version of Chai they are compatible with.

> The best way to determine what your peer dependency requirements should be is to actually follow semver. Assume that only changes in the host package's major version will break your plugin. Thus, if you've worked with every 1.x version of the host package, use "~1.0" or "1.x" to express this. If you depend on features introduced in 1.5.2, use ">= 1.5.2 < 2".